pkg/registry/core/service/allocator/bitmap.go

```diff
@@ -205,3 +220,40 @@ func (rss randomScanStrategy) AllocateBit(allocated *big.Int, max, count int) (i
 }
 
 var _ bitAllocator = randomScanStrategy{}
```

Comment:
Is randomScanStrategy still used anywhere?

Reply:
If you really want to be cautious, you could make the gate flip between randomScanStrategy and randomScanStrategyWithOffset, so nobody hits any bugs you introduced, otherwise there's not much point in keeping randomScanStrategy code